go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift-online/ocm-cli
 
-go 1.18
```

Comment:
We should follow the pattern ROSA folks suggested. Keeping this at 1.18 and if an go bump is required, use a separate MR.

Reply:
Makes sense. Reverted the version to as it is